connector-runtime/src/main/java/io/camunda/connector/runtime/inbound/event/MessageInboundTarget.java

```diff
@@ -0,0 +1,99 @@
+package io.camunda.connector.runtime.inbound.event;
+
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import io.camunda.connector.api.inbound.InboundConnectorResult;
+import io.camunda.connector.api.inbound.InboundConnectorTarget;
+import io.camunda.connector.api.inbound.InboundConnectorWrapperException;
+import io.camunda.zeebe.client.ZeebeClient;
+import io.camunda.zeebe.client.api.response.PublishMessageResponse;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MessageInboundTarget extends InboundConnectorTarget {
+  private static final Logger LOG = LoggerFactory.getLogger(MessageInboundTarget.class);
+
+  private final String messageName;
```

Comment:
I wonder if it make sense to have the message properties here and the startEvent properties in the super class - should be both in subclasses I guess?

That's said maybe we should have `AbstractMEssageInboundTarget` with those properties in the SDK and only the "triggerEvent" part here? Same for StartEvent

Reply:
I reorganized the whole class hierarchy here & in the SDK, I hope it feels a bi more robust now 🙂